File: xrecycleviewlib/src/main/java/com/ashlikun/xrecycleview/BaseSwipeInterface.java
package com.ashlikun.xrecycleview;

/**
 * Created by Administrator on 2016/8/8.
 */

public interface BaseSwipeInterface {
    void setRefreshLayout(RefreshLayout refreshLayout);

    public RefreshLayout getRefreshLayout();

    void setOnLoaddingListener(OnLoaddingListener swipeRefreshLayout);

    PageHelp getPageHelp();


    PageHelpListener getPageHelpListener();
}

File: xrecycleviewlib/src/main/java/com/ashlikun/xrecycleview/BaseSwipeInterface.kt
package com.ashlikun.xrecycleview

/**
 * Created by Administrator on 2016/8/8.
 */
interface BaseSwipeInterface {
    val refreshLayout: RefreshLayout?
    val onLoaddingListener: OnLoaddingListener?
    val pageHelp: PageHelp
    val pageHelpListener: PageHelpListener?
}